ush/python/pygfs/task/snowens_analysis.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from typing import Dict, List, Any
+from pprint import pformat
+import numpy as np
+from netCDF4 import Dataset
+
+from wxflow import (AttrDict,
+                    FileHandler,
+                    to_fv3time, to_YMD, to_YMDH, to_timedelta, add_to_datetime,
+                    rm_p, chdir,
+                    parse_j2yaml, save_as_yaml,
+                    Jinja,
+                    logit,
+                    Executable,
+                    WorkflowException)
+from pygfs.task.analysis import Analysis
```

Comment:
```suggestion
from pygfs.analysis import Analysis
```

Reply:
I'm not sure why (yet) but while the first suggestion above worked, this one does not seem to.

Reply:
Never mind, neither work, I'm going to not remove `task` until this is sorted out, I think it's outside the scope of this PR.